### PR TITLE
Updates native mobile resources

### PR DESCRIPTION
**Pull request by jdiehl**
Fixed widget urls
Added missing widgets
Add nanoflow actions

### CODE REVIEW

**Comment by jdiehl** [comment]
@ConnorLand

File: content/en/docs/appstore/modules/native-mobile-resources.md [hunk 1]
```diff
@@ -27,37 +27,159 @@ These are the native widgets included in these resources along with links to the
...
+    * tag: "networking"
+    * order: "newest" 
+* **Register deep link** – registers a nanoflow to handle all deep link requests
+
+### 3.6 Native Network Category
+
+These are the nanoflow actions in the native network category:
+
+* **Is cellular connection** – checks if the device is connected to the cellular network
+* **Is connected** – checks if the device is connected
```
Comment (ConnorLand) [inline content/en/docs/appstore/modules/native-mobile-resources.md]: Connected to what?

File: content/en/docs/appstore/modules/native-mobile-resources.md [hunk 2]
```diff
@@ -27,37 +27,165 @@ These are the native widgets included in these resources along with links to the
...
+    * Model: the device model (incomplete, use DeviceId instead)
+    * ReadableVersion: the application version and build number
+    * SystemName: the operating system name (iOS or Android)
+    * SystemVersion: the operating system version
+    * Timezone: the time zone of the device
+    * TotalDiskCapacity: the total storage capacity of the device in bytes
+    * TotalMemory: the total memory capacity of the device in bytes
+    * UniqueId: a unique identifier for the device
+    * UserAgent: the user agent of the device's web browser
+    * Version: the application version as set in Build Native App
```
Comment (ConnorLand) [inline content/en/docs/appstore/modules/native-mobile-resources.md]: Do instances of "Build Native App" refer to the Native Mobile Builder UI tool?

File: content/en/docs/appstore/modules/native-mobile-resources.md [hunk 3]
```diff
@@ -27,37 +27,165 @@ These are the native widgets included in these resources along with links to the
...
+    * BuildNumber: the build number as set in Build Native App
+    * BundleId: the application bundle identifier as set in Build Native App
+    * Carrier: name of the network operator
+    * DeviceCountry: the country of the device
+    * DeviceId: an identifier for the device model (for example: iPhone7)
+    * DeviceLocale: the locale of the device
+    * FontScale: the device font scale (1 is normal)
+    * FreeDiskStorage: the available storage size in bytes
+    * Manufacturer: the device manufacturer (for example: Apple)
+    * Model: the device model (incomplete, use DeviceId instead)
```
Comment (ConnorLand) [inline content/en/docs/appstore/modules/native-mobile-resources.md]: What does "incomplete, use DeviceID instead" mean?

File: content/en/docs/appstore/modules/native-mobile-resources.md (hunk 1 above)
Comment (jdiehl) [reply content/en/docs/appstore/modules/native-mobile-resources.md]: the Internet

File: content/en/docs/appstore/modules/native-mobile-resources.md [hunk 4]
```diff
@@ -27,37 +27,165 @@ These are the native widgets included in these resources along with links to the
...
+| [Signature](https://github.com/mendix/native-widgets/tree/main/packages/pluggableWidgets/signature-native) | Creates a canvas on which a user can draw. The drawing will be saved as a *base64* string. |
+| [Slider](https://github.com/mendix/native-widgets/tree/main/packages/pluggableWidgets/slider-native) | Changes a numeric value using a slider. |
+| [Switch](https://github.com/mendix/native-widgets/tree/main/packages/pluggableWidgets/switch-native) | Changes a Boolean value using a switch. |
+| [Toggle Buttons](https://github.com/mendix/native-widgets/tree/main/packages/pluggableWidgets/toggle-buttons-native) | Changes an enumeration value using a group of buttons. |
+| [Video Player](https://github.com/mendix/native-widgets/tree/main/packages/pluggableWidgets/video-player-native) | Plays a video loaded from a URL. |
+| [Web View](https://github.com/mendix/native-widgets/tree/main/packages/pluggableWidgets/web-view-native) | Displays an external web page or loads custom HTML. |
+
+## 3 Nanoflow Actions
+
+SOME EXPLANATION AND OR CROSSLINKS HERE.
```
Comment (ConnorLand) [inline content/en/docs/appstore/modules/native-mobile-resources.md]: Is there anything in particular we want to say about nanoflow actions to guide developers?

File: content/en/docs/appstore/modules/native-mobile-resources.md (hunk 3 above)
Comment (jdiehl) [reply content/en/docs/appstore/modules/native-mobile-resources.md]: Models are maintained by "a community" and it is unclear which models are maintained and which are not. DeviceId contains the same information but is more reliable. So recommendation is to use that instead.

File: content/en/docs/appstore/modules/native-mobile-resources.md [hunk 5]
```diff
@@ -27,37 +27,165 @@ These are the native widgets included in these resources along with links to the
...
+| [Switch](https://github.com/mendix/native-widgets/tree/main/packages/pluggableWidgets/switch-native) | Changes a Boolean value using a switch. |
+| [Toggle Buttons](https://github.com/mendix/native-widgets/tree/main/packages/pluggableWidgets/toggle-buttons-native) | Changes an enumeration value using a group of buttons. |
+| [Video Player](https://github.com/mendix/native-widgets/tree/main/packages/pluggableWidgets/video-player-native) | Plays a video loaded from a URL. |
+| [Web View](https://github.com/mendix/native-widgets/tree/main/packages/pluggableWidgets/web-view-native) | Displays an external web page or loads custom HTML. |
+
+## 3 Nanoflow Actions
+
+SOME EXPLANATION AND OR CROSSLINKS HERE. 
+
+For more information, see the JavaScript nanoflow actions [source code](https://github.com/mendix/native-widgets/tree/main/packages/jsActions/nanoflow-actions-native/src) in GitHub.
```
Comment (ConnorLand) [inline content/en/docs/appstore/modules/native-mobile-resources.md]: I put this in, but if I should delete the sentence just let me know.

File: content/en/docs/appstore/modules/native-mobile-resources.md (hunk 2 above)
Comment (jdiehl) [reply content/en/docs/appstore/modules/native-mobile-resources.md]: I mean NBUI. Please use the common term for that.

File: content/en/docs/appstore/modules/native-mobile-resources.md (hunk 4 above)
Comment (jdiehl) [reply content/en/docs/appstore/modules/native-mobile-resources.md]: Not really, I think they are pretty self-explanatory

File: content/en/docs/appstore/modules/native-mobile-resources.md (hunk 5 above)
Comment (jdiehl) [reply content/en/docs/appstore/modules/native-mobile-resources.md]: This is good, thanks

**Review by ConnorLand — APPROVED** [review]

**Pull request merged**